src/presets/genezio/utils.ts

```diff
@@ -0,0 +1,53 @@
+import fsp from "node:fs/promises";
+import type { Nitro } from "nitropack/types";
+import { dirname, resolve } from "pathe";
+export async function generateFunctionFiles(nitro: Nitro) {
+  const name = await getNameFromPackage();
+  const initialPath = dirname(nitro.options.output.dir);
+  const genezioConfigPath = resolve(initialPath, "genezio.yaml");
+  const genezioConfigContent = `
```

Comment:
Normally zero config nitro presets use a special "output directory" that is supported by deployment providers instead of writing to project root (which is version controlled)

If manual file is required, we should keep it as docs only (i would be happy to explain this more if you like to integrate zero deploy with your platform)

Reply:
(moved to the docs for now)